src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/project.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "",
+  "authors": [ "" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+
+  "dependencies": {
+  },
+
+  "commands": {
+    "WCFClientStressTestsCore" : "WCFClientStressTestsCore"
+  },
+
+  "frameworks" : {
+    "dnxcore50" : {
+      "dependencies": {
+        "Microsoft.Test.Stress":  "1.0.0",
+        "System.Console": "4.0.0-beta-*",
+        "System.Collections": "4.0.10-beta-*",
+        "System.Linq": "4.0.0-beta-*",
+        "System.Threading": "4.0.10-beta-*",
+        "Microsoft.CSharp": "4.0.0-beta-*",
+        "System.Runtime.Extensions": "4.0.10-beta-*",
+        "System.Diagnostics.Debug": "4.0.10-beta-*",
+        "System.Text.Encoding": "4.0.10-beta-*",
+        "System.Collections.Specialized": "4.0.0-beta-*",
+        "System.Diagnostics.Contracts": "4.0.0-beta-*",
+        "System.Linq.Queryable": "4.0.0-beta-*",
+        "System.Net.Http": "4.0.0-beta-*",
+        "System.Net.NameResolution": "4.0.0-beta-*",
+        "System.Net.Security": "4.0.0-beta-*",
+        "System.Net.WebHeaderCollection": "4.0.0-beta-*",
+        "System.Reflection.DispatchProxy": "4.0.0-beta-*",
+        "System.Runtime.Serialization.Xml": "4.0.10-beta-*",
+        "System.Runtime.Serialization.Primitives": "4.0.10-beta-*",
+        "System.Security.Cryptography.Hashing": "4.0.0-beta-*",
+        "System.Xml.XmlDocument": "4.0.0-beta-*",
+        "System.ServiceModel.Http": "4.0.10-beta-*",
+        "System.ServiceModel.NetTcp": "4.0.0-beta-*",
+        "System.ServiceModel.Primitives": "4.0.0-beta-*"
```

Comment:
Jason has recently updated all the project.json files to use only the dependencies actually used.  Please look at the other test project.json files and see if you can reduce yours to match.  Incidentally, once you build, I expect it to generate a project.lock.json file that will need to be checked in.  Make sure the "locked" property inside that file is set to "true" like the others